app/src/main/java/com/protify/protifyapp/FirestoreHelper.kt

```diff
@@ -313,30 +363,36 @@ class FirestoreHelper() {
                             distance = (document.data["distance"] as Long).toInt(),
                             nameLower = document.data["nameLower"].toString(),
                             isOutside = (document.data["isOutside"] as Boolean),
-                            isOptimized = (document.data["isOptimized"] as Boolean)
+                            isOptimized = (document.data["isOptimized"] as Boolean),
+                            isAiSuggestion = (document.data["isAiSuggestion"] as Boolean),
+                            isUserAccepted = (document.data["isUserAccepted"] as Boolean)
                         )
                         callback(document.id, event)
                     }
                 }
                 //If it finds more than one document, callback null and empty event
                 else {
-                    callback(null, FirestoreEvent(
-                        name = "",
-                        startTime = LocalDateTime.now(),
-                        endTime = LocalDateTime.now(),
-                        location = "",
-                        description = "",
-                        timeZone = "",
-                        importance = 0,
-                        attendees = null,
-                        rainCheck = false,
-                        isRaining = false,
-                        mapsCheck = false,
-                        distance = 0,
-                        nameLower = "",
-                        isOutside = false,
-                        isOptimized = false
-                    ))
+                    callback(
+                        null, FirestoreEvent(
+                            name = "",
+                            startTime = LocalDateTime.now(),
+                            endTime = LocalDateTime.now(),
+                            location = "",
+                            description = "",
+                            timeZone = "",
+                            importance = 0,
+                            attendees = null,
+                            rainCheck = false,
+                            isRaining = false,
+                            mapsCheck = false,
+                            distance = 0,
+                            nameLower = "",
+                            isOutside = false,
+                            isOptimized = false,
+                            isAiSuggestion = false,
+                            isUserAccepted = false
+                        )
+                    )
```

Comment:
Is there a way to make these nullable so we don't need to pass every variable? something with so many variables is a little scary to look at